ui/app/models/gcp/config.js

```diff
@@ -65,8 +69,43 @@ export default class GcpConfig extends Model {
     'identityTokenTtl',
   ];
 
+  get isWifPluginConfigured() {
+    return !!this.identityTokenAudience || !!this.identityTokenTtl || !!this.serviceAccountEmail;
+  }
+
+  isAccountPluginConfigured = false;
+  // the "credentials" param is not checked for "isAccountPluginConfigured" because it's never return by the API
+  // additionally credentials can be set via GOOGLE_APPLICATION_CREDENTIALS env var so we cannot call it a required field in the ui.
+  // thus we can never say for sure if the account accessType has been configured so we always return false
```

Comment:
In the future, I think it's helpful for comments to be before the thing they're describing. No need to change as that will make the approval stale, but just as a future suggestion